pubspec.yaml

```diff
@@ -1,6 +1,6 @@
 name: ardrive_io
 description: A new Flutter package project.
-version: 1.0.0
+version: 1.2.0
```

Comment:
Is this correct? Instead of v1.1.0?

Reply:
Yes. The last release we didn't update the version